mllib/src/main/scala/org/apache/spark/ml/classification/ProbabilisticClassifier.scala

```diff
@@ -245,6 +245,13 @@ private[ml] object ProbabilisticClassificationModel {
         v.values(i) /= sum
         i += 1
       }
+    } else {
+      var i = 0
+      val size = v.size
+      while (i < size) {
+        v.values(i) = 1.0 / size
```

Comment:
Is Array.fill faster here?

Reply:
But `Array.fill` will create a new array...

Reply:
You could use `java.util.Arrays.fill` to update `v.values` in-place, but I'm not sure that it'll make a huge difference.